style/style.css

```diff
@@ -87,6 +85,268 @@ header a {
+}
+
+.content ul {
+  margin-bottom: 1rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.content ul li {
+  padding: 0.5rem;
+  border: 0.5px solid #8993a4;
+}
+
+.btn {
+  background-color: #ff6b00;
+  padding: 12px;
+  color: white;
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  justify-self: start;
+}
+
+.btn:hover {
+  background-color: #fff5e1;
+  border: 1px solid #ff6b00;
+  color: #ff6b00;
+}
+
+.btn:active {
+  background: #e05304;
+  color: white;
+}
+
+.project {
+  width: 100%;
+}
+
+.card {
+  display: grid;
+  grid-template-columns: 1fr;
+  align-items: center;
+  width: 100%;
+  background: url("/images/ImgPlaceholder2.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin-bottom: 3rem;
+}
+
+.card-content {
+  margin-left: 1.2rem;
+}
+
+.card h2 {
+  font-family: "Crete Round", sans-serif;
+  font-size: 36px;
+  font-weight: 400;
+  text-align: left;
+  margin: 5rem 0 1rem;
+  color: white;
+}
+
+.card p {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 24px;
+  color: #fff;
+  margin-bottom: 1rem;
+}
+
+.card ul {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  gap: 1rem;
+}
+
+.card ul li {
+  color: white;
+  padding: 1rem;
+  background: rgba(255, 255, 255, 0.24);
+  border: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+}
+
+.card button {
+  width: 100%;
+  padding: 1rem;
+  font-family: "Inter", sans-serif;
+  font-size: 17px;
+  font-weight: 700;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  color: white;
+  background: #ff6b00;
+  margin-top: 1rem;
+}
+
+.about-section {
+  height: 50rem;
+  background-image: url(/images/Frame.png);
+  background-size: 45%;
+  background-repeat: no-repeat;
+  background-color: #1c1a19;
+  background-position: bottom right;
+}
+
+.about-me {
+  height: 50rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: left;
+  gap: 1rem;
+}
+
+.about-me h2 {
+  font-family: "Crete Round", sans-serif;
+  font-size: 72px;
+  font-weight: 400;
+  letter-spacing: 0.3px;
+  text-align: left;
+  color: #ff6b00;
+  margin: 3rem 0 1rem;
+}
+
+.about-me p {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+  margin-bottom: 1rem;
+}
+
+.btn-about {
+  align-self: flex-start;
+}
+
+.about-section hr {
+  border-bottom: 2px solid rgb(255, 255, 255);
+  width: 80%;
+  margin: 0 auto;
+}
+
+.skills {
+  width: 100%;
+  height: 50rem;
+  background-color: #1c1a19;
+}
+
+.my-stack {
+  display: grid;
+  grid-template-columns: 1fr;
+  align-items: center;
+  background-color: #1c1a19;
+  color: white;
+  margin-top: 4rem;
+}
+
```

Comment:
- [ ] Great jon 👍 adding the technology categories ("Languages", "Frameworks" and "Skills") as requested by the previous reviewer however the are still a few changes you need to make
1. - [ ] Please make sure to align the content of the section using `flexbox` as stated in the [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements):

> - Make sure the content of the "About myself" section is aligned with flexbox.

2. - [ ] Please make sure the tech categories section doesn't scroll vertically as it seems unnecessary and hinders use experience. Inner scrollbars can confuse users, and are especially frustrating due to mousewheel behavior. For mobile users, it's confusing, especially if there's no good indication that an inner area scrolls. For users with large/tall screens, having small scrolling areas can be very frustrating and seem overly restrictive restrictive.

     Generally they're a sign that either:

    - your design is very 'busy' (perhaps necessarily)
    
    - you have content both above and below a variably-sized area, and you want to make sure the user is able to reach / see both at once
    
    - the design is trying to avoid putting things 'below the fold' (perhaps unnecessarily), or
    
    - you're avoiding non-square scrolling areas (these used to be hard to do on the web)


https://user-images.githubusercontent.com/44798044/229289209-4522652e-ffaa-4baa-9c30-223b9911cb85.mov

Reply:
i aligned the content of the section using flexbox